packages/utils/upgrade/src/tasks/upgrade/upgrade.ts

```diff
@@ -9,7 +9,7 @@ import * as f from '../../modules/format';
 import { Version } from '../../modules/version';
 
 import type { UpgradeOptions } from './types';
-import { PluginProject } from '../../modules/project/project';
+import { isPluginProject } from '../../modules/project/project';
```

Comment:
Import order? I'm surprised eslint didn't complain about that :thinking: 

Also, you can shorten the import by removing the last `/project`